addons/paint-skew/userscript.js

```diff
@@ -0,0 +1,136 @@
+import { addons, initialize } from "../paint-snap/compatibility.js";
+
+export default async function ({ addon }) {
+  const paper = await addon.tab.traps.getPaper();
+
+  const addSkew = function () {
+    if (!paper.tool || !paper.tool.boundingBoxTool) return;
+    const ST = paper.tool.boundingBoxTool._modeMap.SCALE.constructor;
+    if (!ST) {
+      return;
+    }
```

Comment:
What would this case mean?
If `paper.tool.boundingBoxTool._modeMap.SCALE` exists, it should have a (truthy) constructor. If it didn't exist, then this line wouldn't be reached and line 8 would have thrown already.